torch/autograd/gradcheck.py

```diff
@@ -1437,6 +1439,15 @@ def gradgradcheck(
     Returns:
         True if all differences satisfy allclose condition
     """
+    assert check_fwd_over_rev or check_rev_over_rev, \
+        "Expected at least one of check_fwd_over_rev or check_rev_over_rev to be True"
+    assert not (check_undefined_grad and not check_rev_over_rev), \
+        "Setting check_undefined_grad=True requires check_rev_over_rev to be True"
+    assert not (check_batched_grad and not check_rev_over_rev), (
```

Comment:
I'm confused by this one. Today the batched for rev over rev is the one that is running. Why are we disallowing it?

Reply:
What do you mean by disallowing it? This is the same check we would have when we enter gradcheck, so this one is a bit redundant actually, but this one here will give users a slightly better error.

Reply:
Ho actually the check is ok. But the message is weird right?

Reply:
Oh right, good catch!